mlflow/tracing/display/display_handler.py

```diff
@@ -46,40 +88,11 @@ def display_traces(self, traces: List[Trace]):
         # this should do nothing if not in an IPython environment
```

Comment:
As mentioned in the PR description, even though this function no longer actually displays anything, I think it makes sense from a developer UX point of view to keep the name as `display_traces()`.